src/screens/home/components/Tip.tsx

```diff
@@ -16,16 +18,18 @@ export const Tip = () => {
         <Box flex={1}>
           <Text>
             <Text fontWeight="bold">{i18n.translate('Home.DiagnosedView.Tip.Title')}</Text>
-            <Text>{i18n.translate(`Home.DiagnosedView.Tip.${region}.Body`)}</Text>
+            <Text>{regionalI18n.translate(`RegionContent.DiagnosedView.Active.${region}.Tip.Body`)}</Text>
           </Text>
         </Box>
       </Box>
       <Box paddingHorizontal="m" paddingTop="s">
         <ButtonSingleLine
-          text={i18n.translate(`Home.DiagnosedView.Tip.${region}.CTA`)}
+          text={regionalI18n.translate(`RegionContent.DiagnosedView.Active.${region}.Tip.CTA`)}
           variant="thinFlatNeutralGrey"
           externalLink
-          onPress={() => Linking.openURL(i18n.translate(`Home.DiagnosedView.Tip.${region}.URL`))}
+          onPress={() =>
```

Comment:
@smcmurtry looks like we missed this one

Reply:
It just shifted down a line: `Linking.openURL(regionalI18n.translate(`RegionContent.DiagnosedView.Active.${region}.Tip.URL`))`